### PR TITLE
fixed bug with endless loop when using delimiter on folded directory

### PR DESCRIPTION
**Pull request by awskii**
For #[588]

### CODE REVIEW

File: openstack/objectstorage/v1/objects/testing/fixtures.go [hunk 1]
```diff
@@ -69,16 +76,25 @@ func HandleListObjectsInfoSuccessfully(t *testing.T) {
         "bytes": 14,
         "name": "goodbye",
         "content_type": "application/octet-stream"
-      },
+      },
```
Comment (jtopjian) [inline openstack/objectstorage/v1/objects/testing/fixtures.go]: nit: It looks like there's an extra space at the end of this line.

File: openstack/objectstorage/v1/objects/testing/fixtures.go [hunk 2]
```diff
@@ -69,16 +76,25 @@ func HandleListObjectsInfoSuccessfully(t *testing.T) {
...
         "content_type": "application/octet-stream"
-      },
+      }, 
       {
         "hash": "451e372e48e0f6b1114fa0724aa79fa1",
         "last_modified": "2016-08-17T22:11:58.602650",
         "bytes": 14,
         "name": "hello",
         "content_type": "application/octet-stream"
+      },{
```
Comment (jtopjian) [inline openstack/objectstorage/v1/objects/testing/fixtures.go]: nit: can you break this onto two lines?

**Comment by awskii** [comment]
Done.

**Comment by jtopjian** [comment]
@awskii I've been able to reproduce this problem outside of a unit test -- thank you for reporting it :)

As you've mentioned, this might be part of a larger issue, which I will investigate. This might take me some time, though, but I'm definitely looking into it.

**Comment by awskii** [comment]
Will PR be merged?

**Comment by jtopjian** [comment]
I need to confirm if this is the appropriate fix or if the root cause is coming from another area of Gophercloud.

**Comment by jrperritt** [comment]
closing in favor of #606

**Pull request closed without merging**